.github/workflows/tests.yml

```diff
@@ -12,11 +12,20 @@ on:
 jobs:
     unit-tests:
         name: Unit Tests
-        runs-on: ubuntu-16.04
+        runs-on: ubuntu-20.04
         steps:
             - name: Checkout code
               uses: actions/checkout@v2
-            - name: Run tests
+            - name: Run unit tests
               run: |
-                  python -m unittest tests
-
+                 make
```

Comment:
This single `make` will run the first Makefile target, which is `test`, which is redundant, iiuc?

Reply:
You're right, I thought I'd removed this. Will update.